src/Chirp.Infrastructure/Repositories/CheepRepository.cs

```diff
@@ -134,6 +134,38 @@ public async Task CreateAuthor(string authorName)
             await _dbContext.SaveChangesAsync(); // Persist the changes to the database
         }
         
+        public async Task DeleteCheep(int cheepId)
+        {
+            var cheep = await _dbContext.Cheeps.FindAsync(cheepId);
+            if (cheep != null)
+            {
+                _dbContext.Cheeps.Remove(cheep);
+                await _dbContext.SaveChangesAsync();
+            }
+        }
+        
+        public async Task DeleteCheepByAuthor(AuthorDTO Author)
```

Comment:
If this method deletes all cheeps from an author, by the name, it should be called `DeleteCheepsByAuthor`. Added a `s` after ´Cheep`.